README.md

```diff
@@ -76,3 +76,13 @@ The rake task will verify the md5 of the data file, if it is changed, it will sh
 ```bash
 "!!! Data file changed. please update GEM VERSION !!!"
 ```
+
+### Troubleshooting Failed GH Actions
+A GH PAT is required to provide write access to the repo from the create-pull-request step.
+When an error similar to this follow is thrown on the GH action it indicates the PAT has expired
+
+```
+remote: Permission to zeptofs/bsb_nz.git denied to github-actions[bot].
+````
+
+Contact SRE to re-generate the bzb_nz_gh_actions PAT on the sa-zp-ops-pipelines service account and update the ACTIONS_GITHUB_TOKEN secret on this repo
```

Comment:
This repo is public. We could remove this comment, but given:
- BSB's aren't recognised as such in NZ (they're just part of account numbers)
- We only use it for internal validation purposes, and
- When I say "we", I mean almost zero public interest/activity

Then we could make the whole repo private (and keep the comment).